src/string/trie.hpp

```diff
@@ -3,47 +3,40 @@
 using namespace std;
 
 struct Trie {
-  private:
     static constexpr int C_SIZE = 26;    // C_SIZE  : 文字の種類数
     static constexpr int C_BEGIN = 'a';  // C_BEGIN : 開始文字
-    int root = 0;
+    static constexpr int ROOT = 0;
     struct Node {
         int child[C_SIZE];  // 子ノードの番号, 存在しなければ-1
```

Comment:
```suggestion
        array<int, C_SIZE> child = {};  // 子ノードの番号, 存在しなければ-1
```
Aho-Corasick法の Node の定義に合わせて `std::array` を使っても良さそうです。`child[]` で十分なので array を使わないのも良いですね。

Reply:
Aho-Corasick法の Node の定義に合わせて `to` としても良さそうです。
Trie "木" であることが分かりやすくするために、`child` と命名したということでしょうか？

Reply:
`child` はarrayに合わせておこうと思います。
命名についてはTrie木を実装したときは木構造を意識して `child` としたのですが、オートマトンと考えると別の命名がいいかなと思いAho-Corasickでは `to` にしました。Trieも　`to` で意味は通じるのでこちらに揃えようと思います。

Reply:
[f9f29e4](https://github.com/YNUCPC/ynu-icpc-library/pull/80/commits/f9f29e4d0c8f79bfdef8f3725af5ab88a10168ac) で対応しました